packages/react/src/FieldSet/FieldSet.test.tsx

```diff
@@ -54,4 +54,36 @@ describe('FieldSet', () => {
 
     expect(ref.current).toBe(component)
   })
+
+  it('renders the provided hint text after the legend', () => {
+    render(<FieldSet legend="Legend" hint="hint text" />)
+
+    const component = screen.getByRole('group', { name: 'Legend (hint text)' })
+
+    expect(component).toBeInTheDocument()
+  })
+
+  it('renders the default hint text after the legend', () => {
+    const { container } = render(<FieldSet legend="Legend" optional={true} />)
+
+    const legend = container.querySelector('legend')
```

Comment:
You've changed the selector for the previous test, but not in all the other tests.

Reply:
Made consistent

Reply:
This still isn't a `getByRole` selector?

Reply:
I've switched to `getByRole` and `name`.